libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -81,7 +81,7 @@ const AP_Param::GroupInfo AC_PosControl::var_info[] = {
     // @Range: 0.5 5
     // @Increment: 0.1
     // @User: Advanced
-    AP_GROUPINFO("_ACC_XY_FILT", 1, AC_PosControl, _accel_xy_filt_hz, POSCONTROL_ACCEL_FILTER_HZ),
+    // AP_GROUPINFO("_ACC_XY_FILT", 1, AC_PosControl, _accel_xy_filt_hz, POSCONTROL_ACCEL_FILTER_HZ),
```

Comment:
we normally don't leave commented out code in place so if it's gone, let's remove this _accel_xy_filt_hz variable completely.